install_scripts/templates/common/docker-install.sh

```diff
@@ -228,13 +228,6 @@ _installDocker() {
             logSuccess "Installed yum-utils"
         fi
         getUrlCmd
-        if [ "$DIST_VERSION_MAJOR" -eq 7 ]; then
-            yum-config-manager --add-repo=http://mirror.centos.org/centos/7/extras/x86_64 || true
-            $URLGET_CMD "https://www.centos.org/keys/RPM-GPG-KEY-CentOS-7" > EXTRAS_KEY
-        else
-            yum-config-manager --add-repo=http://mirror.centos.org/centos/8/extras/x86_64/os || true
-            $URLGET_CMD "https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official" > EXTRAS_KEY
-        fi
         rpm --import EXTRAS_KEY
```

Comment:
if we don't have something to set EXTRAS_KEY won't this fail?

Reply:
Good catch! I have updated code to remove the EXTRAS_KEY import part, since it will not be used, if we are not going to set the repos